baseline-error-prone/src/main/java/com/palantir/baseline/errorprone/DangerousThrowableMessageSafeArg.java

```diff
@@ -56,17 +58,23 @@ public Description matchMethodInvocation(MethodInvocationTree tree, VisitorState
         }
 
         List<? extends ExpressionTree> args = tree.getArguments();
-        if (args.size() != 2) {
-            return Description.NO_MATCH;
-        }
-
         ExpressionTree safeValueArgument = args.get(1);
         if (THROWABLE_MESSAGE_METHOD.matches(safeValueArgument, state)) {
             return buildDescription(tree)
                     .setMessage("Do not use throwable messages as SafeArg values. "
                             + "SafeLoggable.getLogMessage is guaranteed to be safe.")
                     .build();
         }
+        if (ASTHelpers.isCastable(
+                ASTHelpers.getResultType(safeValueArgument),
```

Comment:
Can we match against unsafe args too? I can't see any reason to log them at all wrapped as an Arg

Reply:
That stretches the `DangerousThrowableMessageSafeArg` a bit too far, I don't think it's unsafe to do that, just silly.

I think that belongs in a separate check (potentially capable of suggesting a fix to unwrap the throwable and apply it as the last param)